reference/matrix/batch_dense_kernels.hpp

```diff
@@ -57,9 +61,9 @@ using BatchEntry = gko::batch_dense::BatchEntry<ValueType>;
 
 
 template <typename ValueType>
-void simple_apply(const BatchEntry<const ValueType> &a,
-                  const BatchEntry<const ValueType> &b,
-                  const BatchEntry<ValueType> &c)
+inline void simple_apply(const BatchEntry<const ValueType> &a,
```

Comment:
It's in a header file and is non-local to the file. So if it's not inline and included in more than one file, it will create duplicate definitions and the linker will complain.

Reply:
Ah, right, I forgot about that.